src/app/pages/settings/settings.page.ts

```diff
@@ -32,4 +32,25 @@ export class SettingsPage implements OnInit {
     this.setOption(GameOption.TickTimer, $event.detail.value);
   }
 
+
+  exportCharacter(slot: number) {
+    this.store.selectOnce(data => data).subscribe(data => {
+      const ignoredKeys: string[] = [];
+
+      const charData = data.charselect.characters[slot];
+      const charName = charData.name;
+
+      const saveData = Object.keys(data).filter(key => !ignoredKeys.includes(key)).reduce((acc, key) => {
+        acc[key] = data[key];
+        return acc;
+      }, {} as any);
+
+      const fileName = `${charName}.qivan`;
```

Comment:
this should be .ws

also, if this is copy-pasted, the code should ideally be moved to a service instead